modules/apps/object/object-service/src/main/resources/resource-actions/resource-actions.xml.tpl

```diff
@@ -38,6 +38,7 @@
 			<supports>
 				<action-key>ADD_OBJECT_ENTRY</action-key>
 				<action-key>PERMISSIONS</action-key>
+				<action-key>VIEW</action-key>
```

Comment:
https://github.com/javierdearcos/liferay-portal/commit/c346c27a41d8cf0872ebb44a72be357b621a1d7d#diff-bd58fc68dd9af9e4f882dd3016fff2041dd1d4ddcaff06c92c2ccaa404fddae9 > I don't know anything about this.

Reply:
Oh... Good to know. Thank you!

I supposed that the VIEW should be a default permission. Changing it and rebasing